src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -79,13 +83,14 @@ class ActivityFeed extends React.Component<Props, State> {
         const hasMoreItems = prevFeedItems && currFeedItems && prevFeedItems.length < currFeedItems.length;
         const didLoadFeedItems = prevFeedItems === undefined && currFeedItems !== undefined;
         const hasInputOpened = currIsInputOpen !== prevIsInputOpen;
+        const hasActiveFeedEntryIdChanged = activeFeedEntryId && prevActiveFeedEntryId;
```

Comment:
Should this be `activeFedEntryId && activeFedEntryId !== prevActiveFeedEntryId`?

Reply:
I had that originally but then `scrollToActiveFeedItemOrErrorMessage` bails if `activeFeedEntryId` is falsy so I just took it out. I could go either way though, do you have a preference?

Reply:
My main concern is that the variable name indicates something changed, but we're not really checking for that.